### PR TITLE
OSDOCS-4320: Updating scaling guidance

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://issues.redhat.com/browse/OSDOCS-4320

Version(s):
4.12+

Link to docs preview:
https://54983--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/planning-your-environment-according-to-object-maximums.html

### CODE REVIEW

File: modules/openshift-cluster-maximums-major-releases.adoc [hunk 1]
```diff
@@ -5,89 +5,108 @@
...
+* 60 pods per node; 30 server and 30 client pods (30k total)
+* 57 image streams/ns (11.4k total)
+* 15 services/ns backed by the server pods (3k total)
+* 15 routes/ns backed by the previous services (3k total)
+* 20 secrets/ns (4k total)
+* 10 config maps/ns (2k total)
+* 6 netpol/ns, including deny-all, allow-from ingress and intra-namespace rules
+* 57 builds/ns
+
+The following factors are known to affect cluster workload scaling, positively or negatively, and should be factored into the scale numbers when planning a deployment.  For additional information and guidance, contact your sales representative or Red Hat support.
```
Comment (sheriff-rh) [inline modules/openshift-cluster-maximums-major-releases.adoc:94]: Great opportunity to `link:https://access.redhat.com/support/[Red Hat support]`.

**Review by sheriff-rh — APPROVED** [review]
Excellent, concise, well-worded expectations around scaling guidance. Awesome work as always, Ashley!

File: modules/openshift-cluster-maximums-major-releases.adoc [hunk 2]
```diff
@@ -5,89 +5,108 @@
...
+As an example, 500 worker nodes (m5.2xl) were tested, and are supported, using {product-title} 4.12, the OVN-Kubernetes network plug-in, and the following workload objects:
+
+* 200 namespaces, in addition to the defaults
+* 60 pods per node; 30 server and 30 client pods (30k total)
+* 57 image streams/ns (11.4k total)
+* 15 services/ns backed by the server pods (3k total)
+* 15 routes/ns backed by the previous services (3k total)
+* 20 secrets/ns (4k total)
+* 10 config maps/ns (2k total)
+* 6 netpol/ns, including deny-all, allow-from ingress and intra-namespace rules
```
Comment (mffiedler) [inline modules/openshift-cluster-maximums-major-releases.adoc:91]: netpol-> networkpolicy or network policies

File: modules/openshift-cluster-maximums-major-releases.adoc [hunk 3]
```diff
@@ -5,89 +5,108 @@
...
 | 2,000 ^[1]^
 
 | Number of pods ^[2]^
 | 150,000
-| 150,000
 
 | Number of pods per node
-| 250
-| 500 ^[3]^
+| 2,500 ^[3]^
```
Comment (mffiedler) [inline modules/openshift-cluster-maximums-major-releases.adoc:26]: My understanding is that this result is experimental and we probably don't want to document it as a tested maximum.  I suggest leaving it at 500 and publishing the 2500 number in a blog or the like.   @afcollins, @rsevilla87 , @jtaleric fyi if you have further comments on this.

File: modules/openshift-cluster-maximums-major-releases.adoc [hunk 4]
```diff
@@ -5,89 +5,108 @@
...
+* Number of projects, or namespaces
+* Number of image streams per project
+* Number of builds per project
+* Number of services/endpoints and type
+* Number of routes
+* Number of shards
+* Number of secrets
+* Number of config maps
+* Rate of API calls, or the cluster “churn”, which is an estimation of how quickly things change in the cluster configuration.
+** Pod creation requests per second over 5 minute windows: `sum(irate(apiserver_request_count{resource="pods",verb="POST"}[5m]))`
```
Comment (mffiedler) [inline modules/openshift-cluster-maximums-major-releases.adoc:109]: Prometheus query for pod creation requests...

File: modules/openshift-cluster-maximums-major-releases.adoc [hunk 5]
```diff
@@ -5,89 +5,108 @@
...
+** All API requests per second over 5 minute windows: `sum(irate(apiserver_request_count{}[5m]))`
```
Comment (mffiedler) [inline modules/openshift-cluster-maximums-major-releases.adoc:110]: Prometheus query for all API requests...

File: modules/openshift-cluster-maximums-major-releases.adoc (hunk 3 above)
Comment (rsevilla87) [reply modules/openshift-cluster-maximums-major-releases.adoc:26]: Agree, 2.500 ppn has not been intensively tested w/ complex workloads

Comment (ahardin-rh) [reply modules/openshift-cluster-maximums-major-releases.adoc:26]: As discussed, I will keep this as 500 to be safe. If we want to point to a blog post later, we can do that in a separate PR. Thank you!

**Pull request merged**